crates/stellar-scaffold-cli/src/commands/generate/contract/mod.rs

```diff
@@ -0,0 +1,332 @@
+use clap::Parser;
+use reqwest;
+use serde::Deserialize;
+use std::process::Command;
+
+#[derive(Parser, Debug)]
+pub struct Cmd {
+    /// Clone contract from `OpenZeppelin` examples
+    #[arg(long, conflicts_with_all = ["ls", "from_wizard"])]
+    pub from: Option<String>,
+
+    /// List available contract examples
+    #[arg(long, conflicts_with_all = ["from", "from_wizard"])]
+    pub ls: bool,
+
+    /// Open contract generation wizard in browser
+    #[arg(long, conflicts_with_all = ["from", "ls"])]
+    pub from_wizard: bool,
```

Comment:
I know we decided on this, but it's always a bit suspicious when there are three mutually exclusive flags. Smells like subcommands to me... But I don't want to slow this down, just noting it.

Reply:
I think that makes more sense TBD and allows for potentially adding args to each that would likely be incompatible.